README.md

```diff
@@ -66,14 +66,14 @@ For example, with `wget`:
 
 ```
 cd ~
-wget https://github.com/digitalocean/doctl/releases/download/v1.16.0/doctl-<version>-linux-amd64.tar.gz
+wget https://github.com/digitalocean/doctl/releases/download/v1.17.0/doctl-<version>-linux-amd64.tar.gz
```

Comment:
Gah, missed some when I dealt with `README`. Let's just follow the model I started, i.e.,

```
wget https://github.com/digitalocean/doctl/releases/download/v<version>/doctl-<version>-linux-amd64.tar.gz
```

Reply:
:+1: